opencog/rule-engine/backwardchainer/ActionSelection.cc

```diff
@@ -40,12 +40,12 @@ HandleCounter ActionSelection::distribution()
 	// Generate all beta distributions of all TVs
 	std::vector<BetaDistribution> betas;
```

Comment:
FWIW, a FloatValue is any vector for floats.  So perhaps you want to use that to implement distributional TV's.

Reply:
Also, if you need two different vectors of floats, you can have that too -- a value can be an n-ary tree of strings and floats. Its built in, its got unit tests, it can be saved in the sql database, it works.